vite.config.js

```diff
@@ -0,0 +1,9 @@
+import { defineConfig } from "vite";
+import eslintPlugin from "vite-plugin-eslint";
+
+export default defineConfig({
+  test: {
+    includeSource: ["./test/**/*.js"],
+  },
+  plugins: [eslintPlugin.default()],
+});
```

Comment:
```suggestion
import { defineConfig } from "vitest/config";
import * as eslintPlugin from "vite-plugin-eslint";

export default defineConfig({
  test: {
    includeSource: ["./test/**/*.js"],
  },
  plugins: [eslintPlugin.default()],
});
```

이렇게 vite가 아니라 vitest/config 에서 defineConfig를 가져올 수 있답니다!